src/index.html

```diff
@@ -9,6 +9,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div id="logo">
+        <img src="./images/logo.png" alt="logo image">
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple" class="is-active">APPLE</a></li>
+          <li class="nav__item"><a href="#samsung">SAMSUNG</a></li>
+          <li class="nav__item"><a href="#smartPhones">SMARTPHONES</a></li>
+          <li class="nav__item">
+            <a
+              href="#laptopDesktop"
+              data-qa="hover"
+            >
+            LAPTOPS & COMPUTERS
```

Comment:
let's use `text-transform` to make this text uppercase

https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform